### ISSUE TITLE
Installation completed successfully but site does not work 

### ISSUE THREAD

**Issue by washaqq** [rtCamp/Frappe-Manager#84]
i had Docker errors but fixed it; https://github.com/rtCamp/Frappe-Manager/issues/83 , everything completed successfully;
```
✅ Halting other sites: Done
✅ Ports Check : Passed
✅ Pulling Docker Images: Done
✅ Starting Docker Containers: Done
✅ Creating Site: Done
```
Site not Found;
![image](https://github.com/rtCamp/Frappe-Manager/assets/16055736/9dd3c04f-0624-4481-b49e-6ffe9c78a122)

**System information:**
- `fm --version` : `0.9.0`
- `lsb_release -a` : 
``` 
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 22.04.3 LTS
Release:        22.04
Codename:       jammy
```

- `docker version`: `Docker version 24.0.7, build afdd53b`
- `docker-compose version`:

```
docker-compose version 1.29.2, build unknown
docker-py version: 5.0.3
CPython version: 3.10.12
OpenSSL version: OpenSSL 3.0.2 15 Mar 2022
```

**Log:**
[fm.log](https://github.com/rtCamp/Frappe-Manager/files/13742969/fm.log)

**Comment by Xieyt** [comment]
I checked the logs and I think the site has been created and the site server has started. How are you accessing the site  ?
Can you do a curl and give me the output here ?

**Comment by washaqq** [comment]
I am accessing via web browser from host computer;
![image](https://github.com/rtCamp/Frappe-Manager/assets/16055736/9fc597c8-946a-4280-9161-26537514ec7c)

Curl on terminal;
```
curl http://192.168.1.41
<!doctype html>
<html lang=en>
<title>404 Not Found</title>
<h1>Not Found</h1>
<p>192.168.1.41 does not exist</p>
```

**Comment by Xieyt** [comment]
Since you're trying to access the site from other computer, you'll need to set up a domain IP mapping in the `/etc/hosts` file of your host system for your created site.

**Comment by washaqq** [comment]
> Since you're trying to access the site from other cothis was not a case before, and i did not get it, my host computer is windows, `fm` is on virtual ubuntu, if i could'nt go with ip address how will i go if i mputer, you'll need to set up a domain IP mapping in the `/etc/hosts` file of your host system for your created site.

this was not a case before, and i did not get it, my host computer is windows, and i did as you suggested but this should not be like this, and no it did not worked.
host;
![image](https://github.com/rtCamp/Frappe-Manager/assets/16055736/b9cd90bc-96be-42f7-a0d7-133e9347e9aa)

**Comment by washaqq** [comment]
Any Idea?

**Comment by SujinIn** [comment]
First, Thank you very much this amazing Automating installation solution!

I have same issue.
I had imagine... the reverse proxy or internal DNS. But I don't have development skill...

So, I edit the hosts file in C:\Windows\System32\drivers\etc

My server's IP address is '192.168.0.250', so I wrote it in hosts file.

`192.168.0.250 myerpnext.com`

Now, I can connect to
`http://192.168.0.250/adminer`  and  `http://myerpnext.com/adminer`
`http://192.168.0.250/mailhog`  and  `http://myerpnext.com/mailhog`

But I can not connect to
`http://192.168.0.250`  and  `myerpnext.com`
(404 Not found)

So, I've test in Server. Not host.
I had used links2 CLI (ssh) for connecting test. It works.
I mean When I try connect localhost in server, It works.

`links2 http://myerpnext.localhost`  in ssh (putty), 
I can see the `Home and login to Frappe, Email. Password, show, Forget Passowrd? Login .... and etc`

`links2 http://localhost`
`404 Not found. localhost does not exist`

I allow the 22, 80, 100, 443, 8000, 8080, 9000 port by ufw.

**Comment by SujinIn** [comment]
I'm not an expert, so I'm sorry if I don't know much.
I tested them one by one.
The address accessed from the server is `http://erp.localhost`, and the internal network address of the server is `http://192.168.0.250`
I set up port forwarding and DDNS on my router. open the 80 port.

Now it is possible to connect from the outside using a ddns address such as `http://aaa.router.com`. (This address is an example)

Wow, I confirmed that the connection was working fine via `http://aaa.router.com/mailhog` from an external network.
However, when going to `http://erp.aaa.router.com`, a 404 Not Found page appears.

I hope this helps.

**Comment by Xieyt** [comment]
This issue has been fixed in #113.

To create site accessible outside.
1. Create site with fqdn name such as `frappe.com`.
2. Add hosts entry on your system for that site.

**Comment by meichthys** [comment]
To follow up on this for future readers.. From what I can tell, 

Your site name MUST match the url you are using to access the frappe instance. It seems as though you cannot access the site using the ip address if your site is using something like `example.com`
